### PR TITLE
fix: fixed behavior when NavTab had ellipsis under conditions

### PR DESCRIPTION
**Pull request by Toxer10**
## Fixes

- fixed behavior when NavTab had ellipsis under conditions

### CODE REVIEW

File: src/components/organisms/ActionsPane/ActionsPane.tsx [hunk 1]
```diff
@@ -91,8 +91,33 @@ const ActionsPane = (props: {contentHeight: string}) => {
   const isActionsPaneFooterExpanded = useAppSelector(state => state.ui.isActionsPaneFooterExpanded);
   const kubeconfigPath = useAppSelector(state => state.config.kubeconfigPath);
   const [key, setKey] = useState('source');
+  const [isButtonShrinked, setButtonCollapsedState] = useState<boolean>();
```
Comment (devcatalin) [inline src/components/organisms/ActionsPane/ActionsPane.tsx:94]: Should probably rename these to match each other

File: src/components/organisms/ActionsPane/ActionsPane.tsx [hunk 2]
```diff
@@ -91,8 +91,33 @@ const ActionsPane = (props: {contentHeight: string}) => {
...
+
   const dispatch = useAppDispatch();
 
+  // Could not get the ref of Tabs Component
+  const tabsList = document.getElementsByClassName('ant-tabs-nav-list')[0];
+  const extraButton = useRef<any>();
+
+  const getDistanceBetweenComponents = () => {
+    const tabsListEl = document.getElementsByClassName('ant-tabs-nav-list')[0].getBoundingClientRect();
```
Comment (devcatalin) [inline src/components/organisms/ActionsPane/ActionsPane.tsx:103]: Could the method return an empty array or undefined? Maybe add a check for this before getting the client rectangle

File: src/components/organisms/ActionsPane/ActionsPane.tsx (hunk 1 above)
Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx:94]: Sure!

File: src/components/organisms/ActionsPane/ActionsPane.tsx (hunk 2 above)
Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx:103]: Sure!

File: src/components/organisms/ActionsPane/ActionsPane.tsx (hunk 1 above)
Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx]: Done

File: src/components/organisms/ActionsPane/ActionsPane.tsx (hunk 2 above)
Comment (Toxer10) [reply src/components/organisms/ActionsPane/ActionsPane.tsx]: Done. I check if tabsList is a non-empty array in useEffect

**Review by devcatalin — APPROVED** [review]

**Comment by olensmar** [comment]
@Toxer10 on first initialization of the actionspane I still see the below - once i resize or select another resize it works fine.. 

image

**Comment by Toxer10** [comment]
> @Toxer10 on first initialization of the actionspane I still see the below - once i resize or select another resource it works fine..
> 
> image

**Comment by Toxer10** [comment]
> @Toxer10 on first initialization of the actionspane I still see the below - once i resize or select another resource it works fine..
> 
> image

Working on it

**Review by olensmar — APPROVED** [review]

**Pull request merged**